packages/visual-tests/src/checkbox.visual.spec.js

```diff
@@ -27,7 +27,7 @@ describe('Checkbox', () => {
       ['custom-label', 'focus'],
     ])('%p', async (variant, state) => {
       await global.runSetup(`components-checkbox--${variant}`);
-      await global.page.waitForSelector('html.hydrated');
+      await global.page.waitForSelector('#root');
```

Comment:
here also this is not enough: 
we need
`      await global.page.waitForSelector('#root > scale-checkbox > label');
`